YapDatabase/Extensions/AutoView/YapDatabaseAutoViewTransaction.m

```diff
@@ -555,10 +555,14 @@ - (void)insertRowid:(int64_t)rowid
 			[databaseTransaction getCollectionKey:&another
 			                               object:&anotherObject
 			                             forRowid:anotherRowid];
-			
-			return sortingBlock(databaseTransaction, group,
-			                      collectionKey.collection, collectionKey.key,        object,
-			                            another.collection,       another.key, anotherObject);
+            
+            if (object && anotherObject) {
```

Comment:
Indentation seems off in all these changes.

Reply:
Oops